agents-api/agents/bookmark_importer/tests.py

```
@@ -31,92 +33,94 @@
         assert react_bookmark.folder_path == "Bookmarks bar"
-        
+
         github_bookmark = next((b for b in bookmarks if "github.com" in b.url), None)
         assert github_bookmark is not None
         assert github_bookmark.folder_path == "Development Tools"
-    
+
     def test_detect_browser_type(self, service):
         """Test browser type detection"""
         # Test with Chrome HTML
         soup = service._parse_bookmark_file(CHROME_BOOKMARK_HTML)[0]
```

Ruff (0.13.3)

Local variable `soup` is assigned to but never used

Remove assignment to unused variable `soup`

(F841)